x-pack/test_serverless/api_integration/test_suites/common/index_management/inference_endpoints.ts

```diff
@@ -26,7 +26,8 @@ export default function ({ getService }: FtrProviderContext) {
   let roleAuthc: RoleCredentials;
   let internalReqHeader: InternalRequestHeader;
 
-  describe('Inference endpoints', function () {
+  // FLAKY: https://github.com/elastic/kibana/issues/189464
```

Comment:
nit: This issue seems to be closed. Can we reopen it or create a new one and mention it here?

Reply:
I think it's okay to keep this comment. We are going to un-skip this commit may be after a week. I'll create a separate ticket to track that work.